storage/mysql/log_storage_test.go

```diff
@@ -87,14 +87,24 @@ func checkLeafContents(leaf trillian.LogLeaf, seq int64, rawHash, hash, data, ex
 	}
 }
 
-func setAllowsDuplicates(db *sql.DB, treeID int64, allowDuplicates bool) error {
-	stmt, err := db.Prepare("UPDATE Trees SET AllowsDuplicateLeaves = ? WHERE TreeId = ?")
+// TODO(codingllama): Replace with a GetTree/UpdateTree sequence, when the latter is available.
+func updateDuplicatePolicy(db *sql.DB, treeID int64, duplicatePolicy trillian.DuplicatePolicy) error {
+	dbPolicy := ""
+	for k, v := range duplicatePolicyMap {
+		if v == duplicatePolicy {
+			dbPolicy = k
+			break
+		}
+	}
+	if dbPolicy == "" {
+		return fmt.Errorf("unknown DuplicatePolicy: %s", duplicatePolicy)
+	}
+	stmt, err := db.Prepare("UPDATE Trees SET DuplicatePolicy = ? WHERE TreeId = ?")
```

Comment:
I'd like to get all SQL statements pulled out of code at some point. Not vital in this PR especially as this is temporary.